publisher/publisher_test.go

```diff
@@ -247,6 +249,63 @@ func TestUploadArtifacts(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestUploadArtifacts_cantBeRunInParallel(t *testing.T) {
+	t.Skip("flaky on CI")
```

Comment:
I appreciate the test but if don't provide any value maybe we should delete it.

I would prefer to have it working in CI thou. (IMHO)